features/pakiti/client/configv3.pan

```diff
@@ -0,0 +1,37 @@
+unique template features/pakiti/client/configv3;
+
+variable PAKITI_RPMS ?= if_exists('config/pakiti/client/config');
```

Comment:
ok. So please, refuse the PR, I'll have a look for using metaconfig for pakiti-client v3 and I'll do another PR.